### PR TITLE
Enable cookies

### PR DESCRIPTION
**Pull request by masensio**

### CODE REVIEW

File: src/com/owncloud/android/authentication/AuthenticatorActivity.java [hunk 1]
```diff
@@ -878,18 +912,16 @@ private void checkBasicAuthorization() {
...
-        existenceCheckIntent.putExtra(OperationsService.EXTRA_REMOTE_PATH, "/");
-        existenceCheckIntent.putExtra(OperationsService.EXTRA_USERNAME, username);
-        existenceCheckIntent.putExtra(OperationsService.EXTRA_PASSWORD, password);
-        existenceCheckIntent.putExtra(OperationsService.EXTRA_AUTH_TOKEN, mAuthToken);
-        
-        if (mOperationsServiceBinder != null) {
-            //Log_OC.wtf(TAG, "starting existenceCheckRemoteOperation..." );
-            mWaitingForOpId = mOperationsServiceBinder.queueNewOperation(existenceCheckIntent);
+        // delete the account if the token has changed
+        if (mAction == ACTION_UPDATE_TOKEN || mAction == ACTION_UPDATE_EXPIRED_TOKEN) {
```
Comment (davivel) [inline src/com/owncloud/android/authentication/AuthenticatorActivity.java]: Not sure this is the right place. For ACTION_UPDATE_EXPIRED_TOKEN is fine, but for ACTION_UPDATE_TOKEN... we are removing the former session before having a new valid one; what if it fails?

Need more discussion,

File: src/com/owncloud/android/authentication/AuthenticatorAsyncTask.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    private static boolean SUCCESS_IF_ABSENT = false;
+
+    private Context mContext;
+    private final WeakReference<OnAuthenticatorTaskListener> mListener;
+    protected Activity mActivity;
+
+    public AuthenticatorAsyncTask(Activity activity) {
+        mContext = activity.getApplicationContext();
+        mListener = new WeakReference<OnAuthenticatorTaskListener>((OnAuthenticatorTaskListener)activity);
+        mActivity = activity;
```
Comment (davivel) [inline src/com/owncloud/android/authentication/AuthenticatorAsyncTask.java]: This attributie is not used, and not needed; please remove it.

File: src/com/owncloud/android/authentication/AuthenticatorActivity.java [hunk 3]
```diff
@@ -878,18 +912,16 @@ private void checkBasicAuthorization() {
     }
 
     private void accessRootFolderRemoteOperation(String username, String password) {
-        Intent existenceCheckIntent = new Intent();
-        existenceCheckIntent.setAction(OperationsService.ACTION_EXISTENCE_CHECK);
```
Comment (davivel) [inline src/com/owncloud/android/authentication/AuthenticatorActivity.java]: The constant ```OperationsService.ACTION_EXISTENCE_CHECK``` should be deleted (and the block code it conditions too).

**Comment by davivel** [comment]
Reviewed, but I'll make some changes myself.

**Comment by davivel** [comment]
Reviewed and updated; OK to me; @masensio , please, review yourself (specially my last commit)

**Comment by masensio** [comment]
Reviewed.  :+1:

**Pull request merged**